test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -2,63 +2,197 @@
 pragma solidity 0.8.15;
 
 import {Test, console2} from "forge-std/Test.sol";
-
+import {EnumerableSet} from "openzeppelin-contracts/contracts/utils/structs/EnumerableSet.sol";
 import {IVotingToken} from "src/interfaces/IVotingToken.sol";
 import {VotingTokenConcrete} from "test/VotingTokenConcrete.sol";
 import {FranchiserFactory} from "src/FranchiserFactory.sol";
 import {Franchiser} from "src/Franchiser.sol";
 
 contract FranchiserFactoryHandler is Test {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
     FranchiserFactory public factory;
     Franchiser public franchiser;
+    VotingTokenConcrete public votingToken;
+
+    mapping(bytes32 => uint256) public calls;
```

Comment:
Named params would be clearer here

Reply:
Done: 5ab7405ae8e5300d5463faa519a6756c3d95c563